resources/metadata/abapAddonAssemblyKitRegisterPackages.yaml

```
@@ -0,0 +1,60 @@
+metadata:
+    name: abapAddonAssemblyKitRegisterPackages
+    description: This step uploads the SAR archives and creates physical Delivery Packages in in the File Content Management System of SAP.
+    longDescription: |
+      This step takes the list of Software Component Versions from the addonDescriptor in the commonPipelineEnvironment.
+      For Packages in status "P"lanned it uploads the SAR archive with the data file and metadata XML of the Delivery Packages composed and exported in the Final Assembly system.
+      and creates physical Delivery Packages. The Packages ares saved in the File Content Management System of SAP.
+      The new status "L"ocked is written back to the addonDescriptor in the commonPipelineEnvironment.
+
+spec:
+  inputs:
+    secrets:
+      - name: abapAddonAssemblyKitCredentialsId
+        description: Credential stored in Jenkins for Addon Assembly Kit as a Service System (AAKaaS)
+        type: jenkins
+    params:
+      - name: abapAddonAssemblyKitEndpoint
+        type: string
+        description: Base URL to the Addon Assembly Kit as a Service System (AAKaaS)
+        scope:
+        - PARAMETERS
+        - STAGES
+        - STEPS
+        - GENERAL
+        mandatory: true
+      - name: username
+        type: string
+        description: User for the Addon Assembly Kit as a Service System (AAKaaS)
+        scope:
+        - PARAMETERS
+        - STAGES
+        - STEPS
+        mandatory: true
+        secret: true
+      - name: password
+        type: string
+        description: Password the Addon Assembly Kit as a Service System (AAKaaS)
+        scope:
+        - PARAMETERS
+        - STAGES
+        - STEPS
```

```suggestion
          - PARAMETERS
```

(I just noticed that other steps already do this as well, but in theory, the user should not be encouraged to place password in their config.yml.)